src/components/scrollbar.scss

```diff
@@ -0,0 +1,23 @@
+.visible-scrollbar, .invisible-scrollbar, .mostly-customized-scrollbar {
```

Comment:
where are these used? I couldn't find them anywhere else in the code but maybe I didn't search it right?

Reply:
Oh right, this was a test document from waayy early on. It's not being used anymore but I just forgot to delete it